### PR TITLE
BPI-145 & BPI-62 – Assets and tags.

### PR DESCRIPTION
**Pull request by guddo**

### CODE REVIEW

**Comment by guddo** [comment]
Here you go @rimi-itk

File: app/config/config.yml [hunk 1]
```diff
@@ -113,13 +113,13 @@ doctrine_mongodb:
...
-    script_dir_name: "%kernel.root_dir%/scripts"
-    name: "Bpi APIbundle MongoDB Migrations"
-    namespace: "Bpi\ApiBundle\Migrations\MongoDB"
+#mongo_db_migrations:
+#    collection_name: "migration_versions"
+#    database_name: "bpi"
+#    dir_name: "%kernel.root_dir%/../src/Bpi/ApiBundle/Resources/config/Migrations/MongoDB"
+#    script_dir_name: "%kernel.root_dir%/scripts"
+#    name: "Bpi APIbundle MongoDB Migrations"
+#    namespace: "Bpi\ApiBundle\Migrations\MongoDB"
```
Comment (rimi-itk) [inline app/config/config.yml]: Unused code should be deleted, not just commented out.

**Comment by OIOIOOOI** [comment]
Please note that migrations package has been removed to at least bring the sources to life, so composer related commands are executed.
The migrations package - https://packagist.org/packages/antimattr/mongodb-migrations-bundle - was removed from the repo itself - https://github.com/antimattr/mongodb-migrations-bundle.

File: src/Bpi/ApiBundle/Resources/config/doctrine/Aggregate.Agency.mongodb.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (rimi-itk) [inline src/Bpi/ApiBundle/Resources/config/doctrine/Aggregate.Agency.mongodb.yml]: Superfluous blank line.

File: src/Bpi/ApiBundle/Resources/config/doctrine/ValueObject.Assets.Name.mongodb.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Bpi\ApiBundle\Domain\ValueObject\Assets\Name:
+  type: embeddedDocument
+  fields:
+    Name:
```
Comment (rimi-itk) [inline src/Bpi/ApiBundle/Resources/config/doctrine/ValueObject.Assets.Name.mongodb.yml]: This should be `name` (all lowercase), right? The fields names in other `*.mongodb.yml` files are all lowercase.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
A few comments have been added. Please address and then we're ready for go!

**Review by rimi-itk — APPROVED** [review]
Looks really good. Thanks.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**